### PR TITLE
fix: Provide primary keys for tables (A-C)

### PR DESCRIPTION
**Pull request by candiduslynx**
Part of https://github.com/cloudquery/cloudquery/issues/15381

BEGIN_COMMIT_OVERRIDE
fix: Provide primary keys for tables (A-C) (https://github.com/cloudquery/cloudquery/pull/15389)
BREAKING-CHANGE: Provide primary keys for tables (A-C) (https://github.com/cloudquery/cloudquery/pull/15389). THe following tables are affected:
* `aws_accessanalyzer_analyzer_archive_rules`
* `aws_apprunner_custom_domains`
* `aws_apprunner_operations`
* `aws_athena_work_group_named_queries`
* `aws_athena_work_group_prepared_statements`
* `aws_athena_work_group_query_executions`
* `aws_autoscaling_group_lifecycle_hooks`
* `aws_backup_plan_selections`
* `aws_backup_plans`
* `aws_cloudformation_stack_instance_summaries`
* `aws_cloudformation_stack_resources`
* `aws_cloudformation_stack_set_operation_results`

END_COMMIT_OVERRIDE

### CODE REVIEW

File: plugins/source/aws/resources/services/cloudtrail/trail_advanced_event_selectors.go [hunk 1]
```diff
@@ -0,0 +1,47 @@
+package cloudtrail
```
[the remark this answers is not in the document]
Comment (candiduslynx) [reply plugins/source/aws/resources/services/cloudtrail/trail_advanced_event_selectors.go:1]: yeah, will do

[the remark this answers is not in the document]
Comment (candiduslynx) [reply plugins/source/aws/resources/services/cloudtrail/trail_advanced_event_selectors.go]: a6357a698d9d5a25c8f05998bd40b96cc0cf008a

File: plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go [hunk 2]
```diff
@@ -49,7 +53,7 @@ func fetchCloudformationStackSetOperationResults(ctx context.Context, meta schem
 	operation := parent.Item.(models.ExpandedStackSetOperation)
 	input := cloudformation.ListStackSetOperationResultsInput{
 		OperationId:  operation.OperationId,
-		StackSetName: stackSet.StackSetName,
+		StackSetName: stackSet.StackSetId,
```
Comment (bbernays) [inline plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go:56]: Why is this changing?

Comment (bbernays) [reply plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go:56]: Nvm... I see the comment above

File: plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go [hunk 3]
```diff
@@ -22,24 +22,28 @@ The 'request_account_id' and 'request_region' columns are added to show the acco
 		Transform: transformers.TransformWithStruct(&types.StackSetOperationResultSummary{}),
 		Columns: []schema.Column{
 			{
-				Name:     "request_account_id",
-				Type:     arrow.BinaryTypes.String,
-				Resolver: client.ResolveAWSAccount,
+				Name:       "request_account_id",
```
Comment (bbernays) [inline plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go:25]: I think we are missing a PK field for this table `aws_cloudformation_stack_set_operation_results` I don't see any field from API response in the PK, and the API returns a paginated list of items

Comment (candiduslynx) [reply plugins/source/aws/resources/services/cloudformation/stackset_operation_results.go:26]: 9b563cdbe204d62fa560f1c49ec32085297f9393

**Review by bbernays — APPROVED** [review]

File: plugins/source/aws/resources/services/cloudtrail/trail_advanced_event_selectors.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (candiduslynx) [reply plugins/source/aws/resources/services/cloudtrail/trail_advanced_event_selectors.go]: https://github.com/cloudquery/cloudquery/pull/15397

**Pull request merged**